dashbuilder/dashbuilder-client/dashbuilder-cms-client/src/main/java/org/dashbuilder/client/cms/screen/transfer/DataTransferScreen.java

```diff
@@ -127,4 +142,22 @@ public String getExportFileName() {
     public String getImportFileName() {
         return DataTransferServices.IMPORT_FILE_NAME;
     }
-}
+
+    private void callExportService(DataTransferExportModel dataTransferExportModel) {
+        try {
+            dataTransferServices.call((RemoteCallback<String>) path -> {
```

Comment:
Could this call take noticeable time? If so, consider using `BusyIndicator` here as well.